proposals/2021-02-enable-ibc-transfers/README.md

```diff
@@ -0,0 +1,12 @@
+# Enable IBC Transfers
+
+The Proposal enables transfering and recieving assets using the IC20 protocol on the Cosmos Hub. If this proposal passes, there will be IBC assets available in the Bank module of the Hub and ATOM will be available on Zones connected over IBC.
```

Comment:
```suggestion
The Proposal enables transferring and receiving assets using the ICS20 standard on the Cosmos Hub. If this proposal passes, there will be IBC assets available in the Bank module of the Hub and ATOM will be available on Zones connected over IBC.
```

Reply:
I would say "sending and receiving" ? Transferring can mean both.